packages/console/src/pages/TenantSettings/DevTenantNotification/index.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * @file Since dev tenant does not have the subscription and billing history tabs,
+ * we use this component to render the usage notification for dev tenants.
+ */
+
+import { useContext } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { TenantsContext } from '@/contexts/TenantsProvider';
+import InlineNotification from '@/ds-components/InlineNotification';
+
+type Props = {
+  readonly className?: string;
+};
+
+function DevTenantNotification({ className }: Props) {
+  const { currentTenant } = useContext(TenantsContext);
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+
+  if (!currentTenant) {
+    return null;
+  }
+
+  const {
+    usage: { tokenUsage },
+    quota: { tokenLimit },
+  } = currentTenant;
```

Comment:
nit: If the component is as generic as `DevTenantNotification`, I'd rather let the token usage & limit logic out of the component, and instead passing the notification content in as props.

Reply:
That was the idea of using `DevTenantNotification`. So far we only have this token usage limit notification. In case of any future feature request for usage or subscription-based notification. Should reuse this component, and update the logic internally.